tooling/noir_js/src/input_validation.ts

```diff
@@ -0,0 +1,55 @@
+// Check if all of the input values are correct according to the ABI
+export function validateInputs(inputs: any, abi: any) {
```

Comment:
using any here because the rust code is currently not being annotated correctly to return type information for the Abi -- changing inputs to be an `Object` will cause errors due to us using the abi to index it